src/specVersions/v0.0/RicardianContractProcessorImpl.ts

```diff
@@ -62,19 +59,40 @@ export class RicardianContractProcessorImpl implements RicardianContractProcesso
     this.wrappedHelpers.push(name)
   }
 
+  protected registerHelper(name: string, fn: Handlebars.HelperDelegate) {
+    Handlebars.registerHelper(name, fn)
+  }
+
   public getSpecVersion() {
     return {
       major: 0,
       minor: 0,
     }
   }
 
+  /**
+   * Process the RicardianContractConfig and return a RicardianContract.
+   *
+   * @param config A `RicardianContractConfig` object
+   */
   /**
    * Process the RicardianContractConfig and return a RicardianContract.
    *
    * @param config A `RicardianContractConfig` object
    */
   public process(config: RicardianContractConfig): RicardianContract {
+    const  version = getContractSpecVersion(config)
```

Comment:
small formatting thing, but looks like there is an extra space after `const` here `const  version` -> `const version`